config_src/infra/FMS1/MOM_interp_infra.F90

```diff
@@ -231,7 +231,7 @@ end subroutine time_interp_extern_3d
 
 !> initialize an external field
 integer function init_extern_field(file, fieldname, MOM_domain, domain, verbose, &
-                                   threading, ierr, ignore_axis_atts )
+                                   threading, ierr, ignore_axis_atts, correct_leap_year_inconsistency )
```

Comment:
The addition of this argument in FMS1 should be replicated in FMS2 - we have identical APIs between src-level code and infra, even if it is not needed and does nothing. Otherwise, it won't compile with FMS2.

Reply:
31 characters is awfully long for an argument, even if it is optional.  I think "inconsistency" can be dropped since it's not adding any information about the purpose of the flag here.